Update playbooks2.rst

leucos opened the pull request:
Added doc on encrypt capability of vars_prompt.
Assuming the example in https://github.com/ansible/ansible/blob/devel/examples/playbooks/prompts.yml is still valid.

leucos committed:
Update playbooks2.rst

Added doc on encrypt capability of vars_prompt.
Assuming the example in https://github.com/ansible/ansible/blob/devel/examples/playbooks/prompts.yml is still valid.

dhozac commented:
salt: and salt_size: together don't really make sense. I think this needs to actually get explained more with regard to what they do and what algorithms are available, as well as the requirement for the passlib library to use it.

leucos commented:
Yep, makes sense. I'll dig the code and come back to you later.

leucos committed:
Adds crypt scheme list for vars_prompt

Added more details on encrypt usage for vars_prompt, including a list of
crypt schemes and a ling to the passlib documentation.

leucos committed:
Cosmetic fixes

Fixed typo in passlib URL link
Fixed style in cryp scheme list
Fixed bullet 7 in variable precedence section
Adds statement regarding passlib requirement

leucos commented:
Should be better now.

mpdehaan commented:
Merged, thanks, will be in next docs rebuild.
